docs/tutorials/google/start.ipynb

```diff
@@ -346,6 +346,55 @@
         "print(hw_circuit)"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Among these best practices, we will highlight two important ones.\n",
+        "\n",
+        "First, if your circuit uses more than one qubit, you can map the circuit onto the grid of the quantum processor in a way that accounts for its topology. You can do this as follows:"
+      ],
+      "metadata": {
+        "id": "e-tOIGJD9c9_"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "device_graph = device.metadata.nx_graph\n",
+        "router = cirq.RouteCQC(device_graph)\n",
+        "hw_circuit = router(circuit)\n",
+        "print(hw_circuit)"
+      ],
+      "metadata": {
+        "id": "LQh5D_Mx9hvk"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Second, if your quantum processor is calibrated to do a specific set of gates, you will need to transform your circuit into that gateset. For example, for a CZ gateset, you can do the following.\n",
+        "\n",
+        "Other gatesets and considerations are included in the [best practices guide](https://quantumai.google/cirq/google/best_practices)."
+      ],
+      "metadata": {
+        "id": "3An1Uo6F9vxS"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "gateset = cirq.CZTargetGateset(allow_partial_czs=True)\n",
+        "hw_circuit = cirq.optimize_for_target_gateset(hw_circuit, gateset=gateset)\n",
```

Comment:
Consider using a different variable than reusing hw_circuit.  It's better if the cell is idempotent.  Also, I've definitely gotten myself. into trouble when re-running cells that reuse variable names